prob-testing.js

```diff
@@ -1,5 +1,23 @@
 // LC 2677 - Chunk Array -- S: O(), T: O()
 
+/*
+Nice work so far!
```

Comment:
@elliotvhill I've suggested changes to your LC 2677 solution!

I would encouraged you to:
1. create a new lc-2677 branch on your laptop
2. make the changes to this file and then `git push`
3. create a new pull request add the danielzayas GitHub user as a reviewer
4. I will comment and suggest changes 
5. After it’s passing all the leetcode tests, we will merge it to the main branch